partner_stage/models/res_partner.py

```diff
@@ -14,8 +14,9 @@ def _get_default_stage_id(self):
         )
 
     @api.model
-    def _read_group_stage_id(self, states, domain, order):
-        return states.search([])
+    def _read_group_stage_id(self, stages, domain, order=None):
+        order = order or "sequence, id"
+        return stages.search([], order=order)
```

Comment:
I'm not sure if this change is correct.
According to the definition, the order parameter should be removed because only two parameters are expected: `groups, domain`, not `order`:
https://github.com/odoo/odoo/blob/6caa783991dad7272391ae8ea0e76f32eadc9401/odoo/models.py#L2330

The original order should be preserved, like this:
```suggestion
    def _read_group_stage_id(self, stages, domain):
        order = order or "sequence, id"
        return stages.search([], order=stages._order)
```